<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataManagerWidget</class>
 <widget class="QWidget" name="DataManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit_msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>1081</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_QueryValue">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Query">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>query</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_table_name">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>15</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>table name :</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_table_name">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_QueryKey">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>45</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>491</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="VLayout_Connect" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="VLayout_Info" stretch="1,1,1,1,1,1,1">
      <item>
       <layout class="QHBoxLayout" name="HLayout_1" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_database_type">
          <property name="text">
           <string>database type :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_database_type"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayout_2" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_connectionm_name">
          <property name="text">
           <string>connectionm name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_connectionm_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayout_3" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_database_filename">
          <property name="text">
           <string>database filename :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_database_filename"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayout_4" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_host_name">
          <property name="text">
           <string>host name :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_host_name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayout_5" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>port :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_port"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayout_6" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string>username :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_username"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="HLayout_7" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_password">
          <property name="text">
           <string>password :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_password"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Connect">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Btn_Insert">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>insert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Delete">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>320</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
